kubernetes/config-maps/nginx/error-pages.conf

```diff
@@ -0,0 +1,70 @@
+error_page 400 = @error_400;
```

Comment:
You added `location` for 405, 406 and 415 below but you forgot to add `error_page` for them here.